kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/service/impl/B2BPurchaseOrderSciquestServiceImpl.java

```diff
@@ -241,6 +267,12 @@ public String getCxml(PurchaseOrderDocument purchaseOrder, String requisitionIni
 
         return cxml.toString();
     }
+    
+    protected KualiDecimal calcTaxRate(KualiDecimal preTaxAmount, KualiDecimal taxAmount) {
+        if (taxAmount == null || preTaxAmount == null) { return KualiDecimal.ZERO; }
+        if (KualiDecimal.ZERO.equals(preTaxAmount)) { return KualiDecimal.ZERO; }

```

Comment:
It appears to be exactly how it was written in KFS 3, so I'm guessing this was just a direct copy paste. If you want this changed, I could easily do this in a matter of a few minutes.